.github/workflows/tests.yml

```diff
@@ -0,0 +1,79 @@
+name: tests
```

Comment:
If we are at GHA now, we could get rid rid of .travis.yml.

Reply:
Good catch.